pkgs/development/python-modules/bidict/default.nix

```diff
@@ -28,31 +25,37 @@ buildPythonPackage rec {
     hash = "sha256-WE0YaRT4a/byvU2pzcByuf1DfMlOpYA9i0PPrKXsS+M=";
   };
 
-  nativeBuildInputs = [
+  build-system = [
     setuptools
-  ];
-
-  propagatedBuildInputs = [
-    sphinx
+    wheel
   ];
 
   nativeCheckInputs = [
     hypothesis
-    py
+    pytest
     pytest-xdist
     pytestCheckHook
-    pytest-benchmark
-    sortedcollections
-    sortedcontainers
     typing-extensions
   ];
 
+  pytestFlagsArray = [
+    # Pass -c /dev/null so that pytest does not use the bundled pytest.ini, which adds
+    # options to run additional integration tests that are overkill for our purposes,
+    # and (most problematically) adds the --benchmark-disable option. This option
+    # requires taking a dependency on pytest-benchmark in order to avoid `error:
+    # unrecognized arguments: --benchmark-disable ...`. Rather than taking an
```

Comment:
(As of https://github.com/jab/bidict/commit/f48bf580dc0a49bf74e6085b4d349feeb180d46d I've removed the `--benchmark-*` options from pytest.ini, which will help avoid this once those changes are released.)